index.html

```diff
@@ -3,17 +3,108 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name=viewport content="width=device-width, initial-scale=1.0">
 
   <title>My Resume</title>
 
   <link rel="stylesheet" href="css/index.css">
+  <link href="https://fonts.googleapis.com/css?family=Cedarville+Cursive&display=swap" rel="stylesheet">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <div class="main-container">
+    <!-- Initial Content -->
+    <div class="banner">
+      <img src="/images/arthur-fleck-img.jpg" alt="Picture of Arthur Fleck">
+      <div class="nav-area">
+        <h1>Arthur Fleck</h1>
+        <nav>
+          <a href="#about">About Me</a>
+          <a href="#skills">My Skills</a>
+          <a href="#work">Work History</a>
+          <a href="#contact">Contact</a>
+        </nav>
```

Comment:
I really like how you set your anchor tags to go to the correct section. This is very neat.